dbms/src/Storages/StorageReplicatedMergeTree.cpp

```diff
@@ -193,6 +193,9 @@ StorageReplicatedMergeTree::StorageReplicatedMergeTree(
     shutdown_event(false), part_check_thread(*this),
     log(&Logger::get(database_name + "." + table_name + " (StorageReplicatedMergeTree)"))
 {
+    initMergeSelectSession();
+    merge_selecting_handle = context_.getSchedulePool().addTask("StorageReplicatedMergeTree", [this] { mergeSelectingThread(); });
```

Comment:
The task name is misleading - better name it something like `StorageReplicatedMergeTree::mergeSelectingThread`

Reply:
Fixed in commit 6629b03af9bb4ebf50b2ab94b7c3a965b773907c

Reply:
👍